bindings_ffi/src/message.rs

```diff
@@ -773,6 +997,42 @@ pub fn content_to_optional_body(content: MessageBody) -> Option<FfiDecodedMessag
         MessageBody::WalletSendCalls(wallet_send_calls) => Some(
             FfiDecodedMessageBody::WalletSendCalls(wallet_send_calls.into()),
         ),
+        MessageBody::Intent(intent) => {
+            if let Some(intent) = intent {
```

Comment:
A cool thing you can do here is do `let intent = intent?;` rather than the if/else since the function returns Option.